.travis.yml

```diff
@@ -27,3 +27,4 @@ env:
     # prevent wine popup dialogs about installing additional packages
     - WINEDLLOVERRIDES="mscoree,mshtml="
     - WINEDEBUG="-all"
+    - DEBUG=electron-packager
```

Comment:
Good call, added it, I put it behind a `debug.enabled` check though just to ensure calling `JSON.stringify` on the `opts` wouldn't negatively impact performance or throw an error unless `DEBUG` is set.